sdk/monitor/monitor-query/samples/v1/javascript/package.json

```diff
@@ -25,6 +25,6 @@
   "dependencies": {
     "@azure/monitor-query": "latest",
     "dotenv": "latest",
-    "@azure/identity": "^4.2.1"
+    "@azure/identity": "^4.0.1"
```

Comment:
Why was this changed to use an older version?

Reply:
This is auto-generated